data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -46,6 +66,7 @@ Let's start with a simple expression:
 ```
 
 In OCaml, everything has a value, and every value has a type. The above example says, “`50 * 50` is an expression that has type `int` (integer) and evaluates to `2500`.” Since it is an anonymous expression, the character `-` appears instead of a name.
+The double semicolon `;;` at the end tells the toplevel to evaluate and print the result of the given phrase.
```

Comment:
Should we already state that double semicolon is not going to be needed when compiling files?

Reply:
Probably not because I don't think we should ask the reader to keep that in mind through the first part of the tour. It adds little value to this first part. But it is important to state it somewhere, I have seen FP programmers (unfamiliar with OCaml) using `;;` in `.ml` files so it would be valuable to address it, but I think the best place to do it is in the second part.